eahub/templates/account/login.html

```diff
@@ -1,32 +1,70 @@
-{% extends 'form.html' %}
+{% extends 'base_bs5.html' %}
+
+{% load crispy_forms_field crispy_forms_tags %}
+
+{% block navbar %}
+    {% include 'components/navbar.html' %}
+{% endblock navbar %}
 
 {% block title %}Login{% endblock %}
 
-{% load crispy_forms_tags %}
+{% block content %}
+    <div class="container account-floating-form">
+        <h1>Login</h1>
+
+        {% include 'includes/messages.html' %}
+    
+        <form method="POST" action="{% url 'account_login' %}">
+            {% csrf_token %}
+
+            {% if form.errors %}
+                <div class="form-group mb-4">
+                    {% for error in form.non_field_errors %}
+                        <p class="error">{{ error|escape }}</p>
+                    {% endfor %}
+                    {% for field in form %}
+                        {% for error in field.errors %}
+                            <p class="error">{{ error|escape }}</p>
+                        {% endfor %}
+                    {% endfor %}
+                </div>
+            {% endif %}
+
+            <div class="form-floating">
+                <input type="email"
+                    class="form-control form-control-lg"
+                    name="{{ form.login.name }}"
+                    placeholder="Name"
+                    id="{{ form.login.id }}"
+                    required>
+                <label for="{{ form.login.id }}">Email</label>
+            </div>
 
-{% block form_fields %}
+            <div class="form-floating">
+                <input type="password"
+                    class="form-control form-control-lg"
+                    name="{{ form.password.name }}"
+                    placeholder="Email"
+                    id="{{ form.password.id }}"
+                    required>
+                <label for="{{ form.password.id }}">Password</label>
+            </div>
 
-{{ form.login|as_crispy_field }}
+            {% if form.captcha %}{% crispy_field form.captcha %}{% endif %}
 
-{{ form.password|as_crispy_field }}
+            <div class="fs-5 additional-links">
+                <a href="{% url 'account_reset_password' %}">
+                    Forgot password?
+                </a>
+            </div>
 
-<div class="field">
-  {% if form.non_field_errors %}
-    {% for error in form.non_field_errors %}
-        <div class="alert alert-danger login-danger" id="login-danger">{{ error }}</div>
-    {% endfor %}
-  {% endif %}
-</div>
+            <input class="btn btn-primary btn-lg" type="submit" value="Login">
+        </form>
 
+    </div>
 {% endblock %}
 
-{% block submit%}Log in{% endblock%}
 
-{% block content_after_form %}
-<div class="panel animated pulse">
-  <div class="alert alert-info hub-info">
-      <h2 style="color: white">Login not working?</h2>
-      <p><a href="{% url 'account_reset_password' %}"><i class="fa fa-envelope"></i> Send me a password reset email</a></p>
-  </div>
-</div>
+{% block footer %}
```

Comment:
Footer messed up (same for sign up)  

 
![image](https://user-images.githubusercontent.com/15908208/108322773-c69f0500-71bd-11eb-9718-9923ba14edbe.png)